### PR TITLE
Add-even homework

### PR DESCRIPTION
**Pull request by NikczemnyOkularnik**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: add-even
🏅 Granted 5 XP for @NikczemnyOkularnik

**Comment by ziobron** [comment]
⏱ 3 XP extra for @NikczemnyOkularnik for on-time delivery

File: homework/add-even/addEven.hpp [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+
+    for (auto &el : numbers)
+    {
+        even = (el % 2); //Checking if number is even or not.
+
+        if (even == false)
+        {
+            even_numbers.push_back(el);
+        }
+    }
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:20]: You don't need this. Just check in loop if number is even and sum it.

File: homework/add-even/addEven.hpp [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
+
+    sum = std::accumulate(even_numbers.begin(), even_numbers.end(), 0);
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:22]: If you want to use std::accumulate, you can pass predicate.

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**